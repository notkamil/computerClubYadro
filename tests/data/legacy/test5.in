2
10:00 19:00
10
10:05 1 client1
10:10 2 client1 1
11:00 1 client2
11:15 2 client2 2
17:00 4 client1
17:30 4 client2