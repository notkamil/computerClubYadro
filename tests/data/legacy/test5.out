10:00
10:05 1 client1
10:10 2 client1 1
11:00 1 client2
11:15 2 client2 2
17:00 4 client1
17:30 4 client2
19:00
1 70 06:50
2 70 06:15